src/utils/JobRestAPIUtils.py

```diff
@@ -944,6 +944,112 @@ def get_vc(username, vc_name):
     return ret
 
 
+def get_vc_meta(username, vc_name):
+    try:
+        if has_access(username, VC, vc_name, ADMIN):
+            result = {}
+            with DataHandler() as data_handler:
+                meta = json.loads(data_handler.GetVC(vc_name)["metadata"])
+                result["scheduling_policy"] = walk_json(meta,
+                                                        "admin",
+                                                        "scheduling_policy",
+                                                        default="RF")
+                result["job_max_time_second"] = walk_json(
+                    meta, "admin", "job_max_time_second")
+                result["interactive_limit"] = walk_json(meta, "admin",
+                                                        "interactive_limit")
+                return result, 200
+        else:
+            return {
+                "error":
+                    "%s do not have permission to query meta from vc %s" %
+                    (username, vc_name)
+            }, 401
+    except Exception as e:
+        logger.exception("Exception in get_vc_meta VC %s for user %s", vc_name,
+                         username)
+
+        return {
+            "error":
+                "failed to get meta from vc %s, exception %s" %
+                (vc_name, str(e))
+        }, 500
+
+
+def patch_vc_meta(username, vc_name, vc_meta):
+    try:
+        if has_access(username, VC, vc_name, ADMIN):
+            if vc_meta is None or len(vc_meta) == 0:
+                return {"error": "empty vc_meta"}, 400
+            with DataHandler() as data_handler:
+                meta = json.loads(data_handler.GetVC(vc_name)["metadata"])
+                if meta.get("admin") is None:
+                    meta["admin"] = {}
+
+                if "scheduling_policy" in vc_meta:
+                    scheduling_policy = vc_meta.pop("scheduling_policy")
+                    if scheduling_policy is not None and scheduling_policy not in {
+                            "RF", "FIFO"
+                    }:
+                        return {
+                            "error":
+                                "unknown scheduling_policy %s" %
+                                (scheduling_policy)
+                        }, 400
+                    else:
+                        meta["admin"]["scheduling_policy"] = scheduling_policy
+
+                if "job_max_time_second" in vc_meta:
+                    job_max_time_second = vc_meta.pop("job_max_time_second")
+                    if job_max_time_second is not None and type(
+                            job_max_time_second) != int:
+                        return {
+                            "error":
+                                "job_max_time_second should be int, got %s" %
+                                (job_max_time_second)
```

Comment:
`"100"` is string, it will print as an integer. Better return the type of the input `job_max_time_second` as well.